src/components/TimeSeriesCard/TimeSeriesCard.story.jsx

```diff
@@ -137,45 +139,6 @@ storiesOf('Watson IoT/TimeSeriesCard', module)
       </div>
     );
   })
-  .add('medium / single line - custom domainRange', () => {
```

Comment:
Did you mean to remove this story?

Reply:
This story and the story you commented below are actually the same. I thought it would be better to have a more distinguishing name as it blended in with the other `size / single line - ` stories. I was having trouble seeing it myself